README.md

```diff
@@ -3,6 +3,7 @@
     <p>palera1n</p>
 </h1>
 <h3 align="center">An iOS 15.0-15.3.1 work-in-progress, tethered checkm8 jailbreak.</h3>
+<h3 align="center">iOS 15.0-15.7.1 work-in-progress, tethered rootless checkm8 jailbreak.</h3>
```

Comment:
Not a big fan of this part... any way to improve this?

Reply:
Sure, I can remove this line. The body already clarified which version is supported

Reply:
Removed the second subheading